draft-keytrans-mcmillion-protocol.md

```diff
@@ -71,9 +71,7 @@ This document uses the TLS presentation language {{!RFC8446}} to describe the
 structure of protocol messages, but does not require the use of a specific
 transport protocol. As such, implementations do not necessarily need to transmit
 messages according to the TLS format and can chose whichever encoding method
-best suits their application. However, cryptographic computations MUST be done
-with the TLS presentation language format to ensure the protocol's security
-properties are maintained.
```

Comment:
Crypto computations do indeed need to be done in TLS format. TLS specifies endianness, that `opaque` fields are length-prefixed byte strings, etc. Not related to ASN.1. See: https://datatracker.ietf.org/doc/html/rfc5246#section-4